src/hotspot/share/gc/parallel/parallelScavengeHeap.hpp

```diff
@@ -81,8 +83,12 @@ class ParallelScavengeHeap : public CollectedHeap {
   // Allocate in oldgen and record the allocation with the size_policy.
   HeapWord* allocate_old_gen_and_record(size_t word_size);
 
+  // in order to provide accurate estimate this method must be called only when the heap has just been collected and compacted
+  inline void capture_live();
```

Comment:
Sentences should start with upper case in the comment. Also I'd prefer to name the method `update_live_estimate()` instead.

Reply:
Done